data/shopify.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "Shopify",
+  "url": "https://www.shopify.com",
+  "career_page_url": "https://www.shopify.com/careers",
+  "type": "Product",
+  "categories": ["B2B"],
```

```suggestion
  "categories": ["cloud_software"],
```

My fault for the wrong suggestion on the other PR